newsroom/push.py

```diff
@@ -421,6 +421,11 @@ def get_coverages(planning_items, original_coverages=[]):
     def get_existing_coverage(id):
         return next((o for o in original_coverages if o['coverage_id'] == id), {})
 
+    def set_text_delivery(coverage, deliveries):
+        if coverage['coverage_type'] == 'text' and deliveries:
+            coverage['delivery_id'] = deliveries[0]
```

Comment:
@mdhaman This should be like `deliveries[0]['item_id']` is that right?

Reply:
yes